web/libs/editor/src/tags/object/Video/Video.js

```diff
@@ -183,6 +187,23 @@ const Model = types
       if (!framerate || Number.isNaN(framerate)) self.framerate = "24";
       else if (framerate < 1) self.framerate = String(1 / framerate);
       else self.framerate = String(framerate);
+
+      // normalize playback speed parameters
+      const defaultPlaybackSpeed = Number(parseValue(self.defaultplaybackspeed, self.store.task?.dataObj));
+      const minPlaybackSpeed = Number(parseValue(self.minplaybackspeed, self.store.task?.dataObj));
```

Comment:
`parseValue()` will break with non-string values
```suggestion
      const data = self.store.task?.dataObj;
      const defaultPlaybackSpeed = Number(parseValue(String(self.defaultplaybackspeed), data));
      const minPlaybackSpeed = Number(parseValue(String(self.minplaybackspeed), data));
```